extensions/binding/org.eclipse.smarthome.binding.dmx/src/main/java/org/eclipse/smarthome/binding/dmx/handler/Lib485BridgeHandler.java

```diff
@@ -0,0 +1,150 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.dmx.handler;
+
+import static org.eclipse.smarthome.binding.dmx.DmxBindingConstants.*;
+
+import java.io.IOException;
+import java.net.Socket;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import org.eclipse.smarthome.binding.dmx.internal.DmxBridgeHandler;
+import org.eclipse.smarthome.binding.dmx.internal.dmxoverethernet.IpNode;
+import org.eclipse.smarthome.binding.dmx.internal.multiverse.Universe;
+import org.eclipse.smarthome.config.core.Configuration;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link Lib485BridgeHandler} is responsible for communication with
+ * an Lib485 instance
+ *
+ * @author Jan N. Klug - Initial contribution
+ */
+
+public class Lib485BridgeHandler extends DmxBridgeHandler {
+    public final static Set<ThingTypeUID> SUPPORTED_THING_TYPES = Collections.singleton(THING_TYPE_LIB485_BRIDGE);
+    public static final int MIN_UNIVERSE_ID = 0;
+    public static final int MAX_UNIVERSE_ID = 0;
+    public static final int DEFAULT_PORT = 9020;
+
+    private final Logger logger = LoggerFactory.getLogger(Lib485BridgeHandler.class);
+    private Map<IpNode, Socket> receiverNodes = new HashMap<IpNode, Socket>();
+
+    public Lib485BridgeHandler(Bridge lib485Bridge) {
+        super(lib485Bridge);
+    }
+
+    @Override
+    protected void openConnection() {
+        if (this.thing.getStatus().equals(ThingStatus.OFFLINE)) {
+            for (IpNode receiverNode : receiverNodes.keySet()) {
+                Socket socket = receiverNodes.get(receiverNode);
+                if (socket == null) {
+                    try {
+                        socket = new Socket(receiverNode.getAddressString(), receiverNode.getPort());
+                    } catch (IOException e) {
+                        logger.warn("Could not connect to {} in {}: {}", receiverNode, this.thing.getUID(), e);
```

Comment:
So if it can be a general connection problem, it is also better to report problems through the Thing status here and reduce logging to debug.